domains/primitives/messenger/src/messages.rs

```diff
@@ -61,6 +61,8 @@ pub struct Channel<Balance, AccountId> {
     /// Owner of the channel
     /// Owner maybe None if the channel was initiated on the other chain.
     pub maybe_owner: Option<AccountId>,
+    /// The amount of fund hold on the owner account for this channel
```

Comment:
I’m not sure if this means “the amount of funds the owner has” or “the amount of funds the owner put on hold”?

```suggestion
    /// The amount of funds put on hold by the owner account for this channel
```